indigo_app/templates/templated_email/task_new_comment.email

```diff
@@ -0,0 +1,12 @@
+{% block subject %}Task #{{ task.pk }} - {{ task.title|safe }}{% endblock %}
+
+{% block html %}
+  {% include 'templated_email/_header.html' %}
+
+  <p>Hello {{ recipient.first_name }},</p>
+
+  <p> {{ comment.user_name }} commented on Task {{ task.pk }} - ({{ task.title|safe }}): <hr> {{ comment.comment }} </p>
```

Comment:
Please make the task a link, like we do here:

https://github.com/laws-africa/indigo/blob/master/indigo_app/templates/templated_email/task_assigned.email#L9